standards/documentation.md

```diff
@@ -0,0 +1,54 @@
+# Documentation Standards
+
+All coding projects MUST include and maintain minimum documentation standards.
+
+## Minimum Standards
+
+Applications MUST include a README
```

Comment:
I would promote that 'Services that rely on other Services MUST include a WIKI in their projects repository that covers:

- Summary and Responsibilities of Service
- Ownership
- Technical Workflow
- Dependent Services
- Error Handling
- API Endpoints (if necessary)

Thoughts @gkallenberg @kfriedman ?